pkg/apis/monitoring/v1/types.go

```diff
@@ -426,6 +432,25 @@ type PrometheusRuleExcludeConfig struct {
 	RuleName string `json:"ruleName"`
 }
 
+// ObjectReference references a PodMonitor, ServiceMonitor, Probe or PrometheusRule object.
+type ObjectReference struct {
+	// Group of the referent.
+	Group string `json:"group"`
```

Comment:
The api-conventions.md doc explains that I think:

"""Although not always necessary to help a controller identify a resource type, “group” is included to avoid ambiguity when the resource exists in multiple groups. It also provides clarity to end users and enables copy-pasting of a reference without the referenced type changing due to a different controller handling the reference."""

Reply:
Yes but I think that in the operator case, it wouldn't make sense for someone to declare this:

```
  excludedFromEnforcement:
    - group: monitoring.example.com
      resource: prometheusrules
      name: foo
      namespace: bar
```

The only value understood by the operator should be `monitoring.coreos.com`. Now we could make the `group` field optional and maybe it would be useful if we ever move away from `monitoring.coreos.com`?

Reply:
agree. group is now optional and defaults to `monitoring.coreos.com`

Reply:
Good call, done.